.buildkite/scripts/run_models.sh

```diff
@@ -19,5 +19,7 @@ dbt deps
 dbt seed --target "$db" --full-refresh
 dbt run --target "$db" --full-refresh
 dbt test --target "$db"
+dbt run --target "$db" --full-refresh
+dbt test --target "$db"
```

Comment:
I know we don't have any incremental logic at the moment, but good to add an incremental test so we can future proof this.

Reply:
added